client/src/services/activity.ts

```diff
@@ -27,10 +27,15 @@ export const checkInGamer = async (activity: Activity) => {
 export const checkOutGamer = async (
   studentNumber: string,
   pcNumber: number,
-) => {
+  execName: string,
+): Promise<boolean> => {
   if (!studentNumber) {
     alert("This table is not occupied.");
-    return;
+    return false;
```

Comment:
what does returning false do?

Reply:
Refresh only if check out actually completes - dont refresh if we encounter an error